wordpress-mobile/WordPress-Android
# Voice to Content: basic media recorder implementation

Fixes: 
https://github.com/Automattic/wordpress-mobile/issues/89
https://github.com/Automattic/wordpress-mobile/issues/90
https://github.com/Automattic/wordpress-mobile/issues/91
https://github.com/Automattic/wordpress-mobile/issues/92

This PR implements a basic version of media recorder and connects it to the "Voice to Content" flow.

-----

## To Test:

- Install and launch the app
- Login into a site in which you have access to create posts
- Navigate to Me > Debug Settings
- Enable voice_to_content and restart the app
- Navigate to My Site
- Tap the FAB
- Tap the microphone
- Give the "Record audio" permission.
- Say something
- Tap the "stop" icon

✅ Verify the microphone disappears, "Assistant Feature Returned Successfully" text is shown quickly, and then the text changes to what you have said.



-----

## Regression Notes

1. Potential unintended areas of impact

    - TODO

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - TODO

3. What automated tests I added (or what prevented me from doing so)

    - TODO

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
